README.md

```diff
@@ -27,7 +27,8 @@ Among the features that this package adds are
 ## Installation
 
 The quickest and easiest way to install is to run 
-`remotes::install_github("ursa-labs/arrowbench")` in R.
+`remotes::install_github("ursa-labs/arrowbench")` in R. If you need to install 
```

Comment:
Should we recommend installing with `dependencies = TRUE` so arrow et al. install?